tests/util/database.py

```diff
@@ -0,0 +1,12 @@
+from cratedb_toolkit.util import DatabaseAdapter
+
+
+def test_quote_ident():
+    database = DatabaseAdapter(dburi="crate://localhost")
+    assert database.quote_ident("my_table") == "my_table"
+    assert database.quote_ident("my-table") == '"my-table"'
+    assert database.quote_ident("MyTable") == '"MyTable"'
+    assert database.quote_ident('"MyTable"') == '"MyTable"'
+    assert database.quote_ident("my_schema.my_table") == "my_schema.my_table"
+    assert database.quote_ident("my-schema.my_table") == '"my-schema".my_table'
```

Comment:
What about this case and friends?
```python
assert database.quote_ident("my_schema"."my_table") == "my_schema.my_table"
```

Reply:
The current logic will not remove such valid quotes, why should it?